Source/Project/ProjectWindow.cpp

```diff
@@ -139,3 +179,6 @@ void ProjectWindow::DragDropMoveRegularFile(std::filesystem::path dir, const ImG
 		std::filesystem::rename(payloadPath.string(), dir.string()+"\\"+payloadPath.filename().string());
 	}
 }
+
+ProjectWindow::ProjectWindow(Scene* scene, AssetHelper* assetHelper, SceneEditor* sceneEditor): _scene(scene), _assetHelper(assetHelper), _sceneEditor(sceneEditor)
```

Comment:
I know this wasn't you but since we're making changes here can we move this constructor to the top of the file